packages/compass-components/src/hooks/use-navigation-tree.spec.tsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
This needs to be removed and replaced with the one that is currently in the databases-navigation package directly